testsuite/MDAnalysisTests/auxiliary/base.py

```diff
@@ -352,6 +352,20 @@ def test_iterate_as_auxiliary_from_trajectory(self, ref):
                          "representative value does not match when iterating through all trajectory timesteps")
         u.trajectory.close()
 
+    def test_auxiliary_read_ts_rewind(self, ref):
+        # add to trajectory
+        u = mda.Universe(COORDINATES_TOPOLOGY, COORDINATES_XTC)
+        u.trajectory.add_auxiliary('test', ref.testdata)
```

Comment:
This could become a fixture (universe with aux file)

Reply:
Done.